docs/source/workflow_outputs.rst

```diff
@@ -513,7 +513,7 @@ Depending on the outputs requested, the file may include:
    ===================================  ==================================================================================================================================
    Type                                 Notes
    ===================================  ==================================================================================================================================
-   Total Consumptions                   Energy use for building total.
+   Total Consumptions                   Energy use for building total and net (i.e., subtracts any power produced by PV or generators).
```

Comment:
This wording seems a bit confusing. 
Maybe add a different line for Net Consumptions. 

```suggestion
Total Consumptions: Total energy used by the building (includes consumption from PV or other onsite generation)
Net Consumptions: Net energy drawn from the grid (excludes consumption from PV or other onsite generation)
```

Reply:
Yeah it's a bit weird, but the documentation as written is more technically correct because you get both total and net energy consumption when you request the "total" timeseries output. Each row in the table is a specific timeseries output "type" that can be quested. For example, in buildstockbatch, you'll get both total and net outputs when you use `include_timeseries_total_consumptions`.

Reply:
Oh, I didn't know they were organized as one row per category.